templates/openproject/files/.platform.app.yaml

```diff
@@ -0,0 +1,20 @@
+name: "app"
+
+type: "ruby:2.6"
+disk: 1024
+
+hooks:
+    build: |
+        bundle install
+        rake assets:precompile
+    deploy: |
+        rails db:migrate
+
+mounts:
+    "/log": "shared:files/log"
+    "/tmp": "shared:files/tmp"
+    "/db": "shared:files/db"
+
+web:
+    commands:
+        start: 'rails server -p $PORT'
```

Comment:
You should not run in production the built-in server. Production use requires an application server such as Unicorn or Puma